python/ray/data/read_api.py

```diff
@@ -2282,6 +2283,9 @@ def from_tf(
 @PublicAPI
 def from_torch(
     dataset: "torch.utils.data.Dataset",
+    parallelism: int = -1,
+    random_split: bool = False,
+    batch_size: int = 32,
 ) -> MaterializedDataset:
     """Create a :class:`~ray.data.Dataset` from a
     `Torch Dataset <https://pytorch.org/docs/stable/data.html#torch.utils.data.Dataset/>`_.
```

Comment:
We need to update the docstring to reflect new changes. Things like this function being inefficient or executing slowly may not be applicable with the new changes.